packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -683,9 +697,13 @@ export class ContainerRuntime
 			runtimeOptions = {},
 			containerScope = {},
 			containerRuntimeCtor = ContainerRuntime,
-			initializeEntryPoint,
 		} = params;
 
+		const initializeEntryPoint =
+			params.initializeEntryPoint ??
+			(async (containerRuntime: IContainerRuntime) =>
+				requestFluidObject(containerRuntime, "/"));
```

Comment:
it might be more accurate if the default object we an IFluidRouter, that just proxies to internal routing, as that better matches the existing behavior:
```
{
get IFluidRouter(){return this;}
async request(req)=>containerRuntime.request(req)
}
```

Reply:
the problem with just returning the default object is that i won't be setup to handle things correctly, as today default objects don't own routing.